packages/dumb/src/components/stats/cbg-utils.tsx

```diff
@@ -26,26 +26,52 @@
  */
 
 import styles from './cbg-colors.css'
+import { BgClasses } from './models'
+
+const CBG_BAR_WIDTH = 234 // Width of the cbg bar in px
```

Comment:
You could mention the unit in the const name (`CBG_BAR_WIDTH_PX`) to make explicit the type of data that is represented here (and to avoid going back to the const definition to know the unit)